servicemesh-install/topics/install-multi-tenant.adoc

```diff
@@ -0,0 +1,77 @@
+[[multi-tenant-install]]
+= Multi-tenant {ProductName} installation
+
+The {ProductName} operator provides support for multi-tenant control plane installations. A multi-tenant control plane is configured so that only specified namespaces can be joined into its {ProductShortName}, isolating the mesh from other installations.
+
+[NOTE]
+====
+You cannot use multi-tenant control plane installations in conjunction with a cluster-wide control plane installation. {ProductName} installations must either be multi-tenant or single, cluster-wide installations.
+====
+
+[WARNING]
+====
+Automatic route creation is currently incompatible with multi-tenant {ProductShortName} installations. Ensure that it is disabled, by setting `ior_enabled` to `false` in your `ServiceMeshControlPlane` if you plan to attempt a multi-tenant installation.
+====
+
+== Known issues
+
+* `MeshPolicy` is still a cluster-scoped resource and applies to all control planes installed in OpenShift. This can prevent the installation of multiple control planes or cause unknown behavior if one control plane is deleted
```

Comment:
```suggestion
* `MeshPolicy` is still a cluster-scoped resource and applies to all control planes installed in OpenShift. This can prevent the installation of multiple control planes or cause unknown behavior if one control plane is deleted.
```